mmdet/models/mask_heads/fcn_mask_head.py

```diff
@@ -136,6 +138,8 @@ def get_seg_masks(self, mask_pred, det_bboxes, det_labels, rcnn_test_cfg,
         if isinstance(mask_pred, torch.Tensor):
             mask_pred = mask_pred.sigmoid().cpu().numpy()
         assert isinstance(mask_pred, np.ndarray)
+        # when enable fp16 training, mask_pred may be float16 numpy array
```

Comment:
`when enabling mixed precision training` or `when mixed precision training is enabled`